src/main/resources/com/prj/MyFirstClass.java

```diff
@@ -5,7 +5,7 @@
 	public static void main(String[] args) {
 		System.out.println("Hi.....");
 		System.out.println("Hi.....");

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)